examples/Workflow/WorkflowWebApp/WorkflowWebApp.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\..\src\Dapr.Workflow\Dapr.Workflow.csproj" />
+  </ItemGroup>
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <LangVersion>latest</LangVersion>
```

Comment:
`10.0`?  (As .NET 6 is compatible only with C# <=10.)

Reply:
Your call. As I understand we have a bit more freedom on choosing newer SDKs/language versions for pluggable components than on the regular SDKs

Reply:
done